analyzers/tests/SonarAnalyzer.UnitTest/TestCases/LockedFieldShouldBeReadonly.CSharp11.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+class Test
+{
+    static readonly object staticReadonlyField = null;
+    static object staticReadWriteField = null;
+
+    readonly object readonlyField = null;
+    object readWriteField = null;
```

Comment:
These do not seem to be unused fields - in case they should be deleted to reduce the noise.